AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts

```diff
@@ -174,8 +229,30 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
                 this.hearing.participants[indexOfJudge].display_name = this.judge.display_name;
             }
         }
-        const text = SanitizeInputText(this.judgeDisplayName.value);
-        this.judgeDisplayName.setValue(text);
+        const text = SanitizeInputText(this.judgeDisplayNameFld.value);
+        this.judgeDisplayNameFld.setValue(text);
+    }
+
+    changeEmail() {
+        let judgeEmail = this.judgeEmail;
+        if (this.judgeEmail === '') {
+            judgeEmail = this.selectedJudgeEmail;
+        }
+        const indexOfJudge = this.hearing.participants.findIndex(x => x.is_judge === true);
```

Comment:
you don't need the `=== true` here, `x.is_judge` is enough

Reply:
i'd also consider just using `find` here, that'll return you an object rather than an index

Reply:
changed